__snapshots__/Slider.md

```diff
@@ -5,9 +5,12 @@
 ```html
 <div id="labelContainer">
     <label for="input" id="label"></label>
-    <div aria-labelledby="label" aria-readonly="true" id="value" role="textbox">
-        10
-    </div>
+    <div
+        aria-labelledby="label"
+        aria-readonly="true"
+        id="value"
+        role="textbox"
+    ></div>
```

Comment:
This points out that is the slider uses the default `10` for a value that the `ariaValueText` prop won't be set at that point. I think being we don't really want `ariaValueText` to reflect that we should make it a programatic getter: ```
get ariaValueText() {
  return this.getAriaValueText(this.value);
}
``` Which would also ensure that the value is always correct without having to be a part of the value setter.

Reply:
Updated in b49eddf